### PR TITLE
adds publishing rules for origin-4.3-kubernetes-1.16.2

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

**Comment by p0lyn0mial** [comment]
I am leaving `origin-4.3-kubernetes-1.16.0-rc.2` rules for now (will remove them in a separate pull). They will also take `master` but that shouldn't be an issue.

File: publishing-rules.yaml [hunk 1]
```diff
@@ -3,6 +3,10 @@ skip-gomod: true
 rules:
 - destination: kubernetes
   branches:
+  - name: origin-4.3-kubernetes-1.16.2
```
Comment (sttts) [inline publishing-rules.yaml]: shouldn't we remove the old one at the same time?

Comment (p0lyn0mial) [reply publishing-rules.yaml]: I wasn't sure if we want that https://github.com/openshift/origin/pull/24016#issuecomment-545872592. Will remove then.

Comment (p0lyn0mial) [reply publishing-rules.yaml]: done

**Pull request merged**